eli5/formatters/features.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+
+class EscapedFeatureName(object):
+    """ Feature name that does not need any additional escaping.
+    """
+    def __init__(self, value):
+        self.value = value
+
+    def __str__(self):
+        return str(self.value)
```

Comment:
Right, I didn't think about Python 2 here, thanks for the catch!
I want it to return text type, so it seems easier to use a `.format()` method instead: ee0c4ae.